src/main/java/frc/robot/subsystems/arm/ArmSubsystem.java

```diff
@@ -12,12 +12,25 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
 import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.robot.subsystems.elevator.ElevatorSubsystem.EvevatorState;
+
 import java.util.function.DoubleSupplier;
 
 public class ArmSubsystem extends SubsystemBase {
   private ArmIO armIO;
 
+  public enum ArmState{
+    GOAL_ANGLE,
+    IDLE
+  }
+
+  public static ArmState armState = ArmState.IDLE;
```

Comment:
you could rename `armState` to just `state` itself. We already know it's the state of the subsystem, and doing `arm.armState` ngl is pretty redundant.

same can be said about the elevator state